curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657e2ecee9f60092c89338d9.md

```diff
@@ -1,28 +1,33 @@
 ---
 id: 657e2ecee9f60092c89338d9
-title: Step 84
+title: Step 85
 challengeType: 0
-dashedName: step-84
+dashedName: step-85
 ---
 
 # --description--
 
-If the user rolls a small straight, then the `Small straight` radio button option should be enabled with a score of `30` points.
+If the user rolls a small straight, then the `Small straight` radio button option should be enabled with a score of `30` points. A small straight consists of four separate dice rolls arranged in numerical order. 
 
-Start by creating an `if` statement to check if `uniqueNumbersStr` is included in the `smallStraightsArr`. If so, call the `updateRadioOption` with `3` and `30` for the arguments. 
+To get started, iterate through the `smallStraightsArr`. You are free to use whichever loop you prefer. If you are using a higher order function like a `forEach` or the `some` method, the callback function should take an argument of `straight`. If you are using a `for...of` loop, then your loop variable should be called `straight`.
```

Comment:
```suggestion
To get started, iterate through `smallStraightsArr` using any loop you prefer. If using a higher-order function like `forEach` or `some`, the callback function should take an argument of `straight`. If using `for...of`, the loop variable should also be `straight`.
```

Reply:
Thank you for grammar suggestions. I'll add those in once you are satisfied with the regex.